README.md

```diff
@@ -26,6 +26,23 @@ Run the `@diplodoc/pdf-generator` command. This will create PDF files right next
 npx -- @diplodoc/pdf-generator@latest -i ./docs-output
 ```
 
+The tool also provides users with flexible customization of headers and footers in the form of HTML format files, the paths to which can be passed through the corresponding options --custom-header and --custom-footer.
+
+By default, the header is not set, but the page numbering in the lower right corner is set in the footer, as shown in the example below.
+
+```
+npx -- @diplodoc/pdf-generator@latest -i ./docs-output
+```
+
+Also below is an example of a command with custom header ([example-custom-header.html](https://github.com/ggerlakh/pdf-generator/blob/master/example-custom-header.html)) and custom footer ([example-custom-footer.html](https://github.com/ggerlakh/pdf-generator/blob/master/example-custom-footer.html)).  
+Even if you specify a custom footer, the numbering still remains in the lower right corner.
+
+```
+npx -- @diplodoc/pdf-generator@latest -i ./docs-output --custom-header ./example-custom-header.html --custom-footer ./example-custom-footer.html
```

Comment:
make a separate section with a table "passed parameters" indicate the default value and what can be passed there and why